prow/pluginhelp/pluginhelp.go

```diff
@@ -41,23 +41,14 @@ type PluginHelp struct {
 	// Description is a description of what the plugin does and what purpose it achieves.
 	// This field may include HTML.
 	Description string
-	// WhoCanUse is a description of the permissions/role/authorization required to use the plugin.
-	// This is usually specified as a github permissions, but it can also be a github team, an
-	// OWNERS file alias, etc.
-	// This field may include HTML.
-	// TODO(qhuynh96): This field is about to be deprecated. WhoCanUse is now under the scope of Command.
-	WhoCanUse string
 	// Usage is a usage string for the plugin. Leave empty if not applicable.
 	// TODO(qhuynh96): This field is about to be deprecated. Usage is now under the scope of Command.
+	// The only plugin that is using this field is releasenote.
 	Usage string
```

Comment:
You can remove this field and move the content of the `releasenote` plugin's `Usage` field to the end of the `PluginHelp.Description` field.

Reply:
I decided not to remove deprecated fields in the PluginHelp yet. As the deck front-end still needs them until it completely shift to the new design.